build.gradle

```
@@ -214,6 +212,16 @@ tasks.register('removeSnapshotSuffix') {
     }
 }
 
+tasks.register('zipSidebarStartPage', Zip) {
+    dependsOn npmInstall
+
+    from('node_modules/@acrolinx/sidebar-startpage/dist/dist-offline/')
```

not sure, but would this task fail if this "from" folder is missing? Or just silently continue, producing an empty zip file? Can we somehow add a check to verify the zip file is not empty?